### ISSUE TITLE
no matter what DNS i use my game wont work

### ISSUE THREAD

**Issue by Villagerjj** [Arisotura/melonDS#676]
im gonna make this as simple as possible:

i have used all available wiimmfi DNS combos and what not
i have a build of it made by someone in a discord. it is meant for wild world (animal crossing DS)
i have gotten 164.132.44.106 with 8.8.8.8 to connect successfully. but i can't get new friend code from copper. (error 20110)
same with 178.62.43.212 and 1.1.1.1 (same error 20110)
no official builds that i tested works with any of the DNSes
yes i have ethernet
yes i have npcap
my custom build (from discord) does not require ethernet (i use it anyway) this fixed the problem with the DNSes.
but like i stated earlier copper won't give me a new friend code. 
game boots normally
need a lot of help

**Comment by MojoJojoDojo** [comment]
Delete the WFC configuration ID from the WFC menu and start this all over it might fix it.

**Comment by windowscj2** [comment]
> Delete the WFC configuration ID from the WFC menu and start this all over it might fix it.

No, it still just won't budge even if I reset it. I use ethernet but still won't let me choose the options for internet

**Comment by windowscj2** [comment]
> Delete the WFC configuration ID from the WFC menu and start this all over it might fix it.

oh wait nevermind
